src/XenobladeChroniclesX/Mods/HUDFreecam/rules.txt

```diff
@@ -0,0 +1,6 @@
+[Definition]
+titleIds = 0005000010116100,00050000101C4C00,00050000101C4D00
+name = "Freecam"
+path = "Xenoblade Chronicles X/Mods/HUD/Freecam"
+description = Look around the world with the developer freecam.|The game functions like normal when you're in this mode. Area renders around camera, so going too far will make your character have no ground and die.||Made by blingbloing, modified by intra.|||Hold 'R' button to move camera up|Hold 'L' button to move camera down|Hold 'X' button to keep camera in place|Hold 'Y' button to speed up camera|Tip: Press and hold the R stick button to speed up even more
```

Comment:
I still think this is too long, did you check the suggestion at https://github.com/ActualMandM/cemu_graphic_packs/pull/567#discussion_r957593867 ? Or what didn't work with it?

Reply:
It looked kinda unpleasant when actually loaded into cemu. This has the flaw of being a really long line in a text editor, but in the cemu graphics pack screen it looks fine

Reply:
You sure? Text wrapping in Cemu is kinda janky so I've always tended to avoid such long texts. But if you really say so.